Combinatorial_map/include/CGAL/Compact_container_with_index.h

```diff
@@ -752,7 +752,10 @@ class Compact_container_with_index
     return false;
   }
 
-  bool owns_dereferencable(const_iterator cit) const
+  bool owns_dereferenceable(const_iterator cit) const
+  { return cit!=end() && owns(cit); }
+
+  CGAL_DEPRECATED bool owns_dereferencable(const_iterator cit) const
```

Comment:
why have the code duplicated and not have `owns_dereferencable` (the old) calling `owns_dereferenceable` (the new)?

Reply:
fixed. Thanks.